### DISCUSSION TITLE
Reading Velocity Model

### DISCUSSION THREAD

**Discussion by kenichi-tsuda** [SPECFEM/specfem3d#1582] [General]
Dear People

My name is Kenichi Tsuda from Japan (working for Shimizu Corporation).
I have one question to  read the velocity model externally.
Originally setting "Model = tomo" in Par_file works to read the velocity model. 
However, the latest version of the code (I have downloaded recently) did not work just setting " Model = tomo" in Par file. 
I'm wondering whether I forgot to change any setting in Par_file.
I'm very appreciated to have any setting to be corrected. 
(kenichi.tsuda@shimz.co.jp is my e-mail address)

Regards

Kenichi Tsuda

**Comment by danielpeter** [comment]
follow up in issue #1583
